### PR TITLE
feat: E2E send test notifications

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**
Tested on Chrome and firefox on macOS

```release-note
NONE
```

### CODE REVIEW

File: e2e-tests/cypress/tests/integration/channels/test_notification/trigger_browser_notification_spec.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+describe('Verify users can receive notification on browser', () => {
+    let offTopic: string;
+
+    before(() => {
+        cy.apiInitSetup({userPrefix: 'other', loginAfter: true}).then(({offTopicUrl}) => {
+            offTopic = offTopicUrl;
+        });
+    });
+
+    it('should be able to receive notification when notifications are enabled on the browser', () => {
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/integration/channels/test_notification/trigger_browser_notification_spec.ts:22]: Do we have test cases in Zephyr? If yes, please add them here, TIA.

File: e2e-tests/cypress/tests/support/notification_commands.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/support/notification_commands.js:1]: Please convert this to typescript.

**Review by saturninoabril — CHANGES_REQUESTED** [review]

File: e2e-tests/cypress/tests/integration/channels/test_notification/trigger_browser_notification_spec.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                    requireInteraction: false,
+                    silent: false,
+                }),
+            );
+        });
+        cy.get('#accountSettingsHeader button.close').click();
+        cy.verifySystemBotMessageRecieved(notificationMessage);
+    });
+
+    it('should not be able to receive notification when notifications are denied on the browser', () => {
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/integration/channels/test_notification/trigger_browser_notification_spec.ts:42]: Thanks for adding MM-T5631! How about his and the next test cases below?

Comment (yasserfaraazkhan) [reply e2e-tests/cypress/tests/integration/channels/test_notification/trigger_browser_notification_spec.ts:42]: Sure Saturn. I'll add MM-T5631_1 as scenarios under 1 test case

File: e2e-tests/cypress/tests/integration/channels/test_notification/trigger_browser_notification_spec.ts
[the remark this answers is not in the document]
Comment (yasserfaraazkhan) [reply e2e-tests/cypress/tests/integration/channels/test_notification/trigger_browser_notification_spec.ts:1]: done added a test case

**Review by saturninoabril — APPROVED** [review]
Thanks @yasserfaraazkhan!

**Pull request merged**